README.md

```diff
@@ -116,6 +116,15 @@ Run `npm run start -- --host=<host>` for a dev server. Navigate to `http://<host
 
 Run `npm update`. This will automatically update package.json and package-lock.json.
 
+When you update a dependency in the package.json, make sure to update the third party licenses file `THIRD-PARTY-LICENSES.csv`. 
+To update, run the command `license-checker  --excludePackages dockstore-ui2@2.3.0 --csv > UPDATED-LICENSES.csv`.
```

Comment:
I think there should a script in the package.json that does this, so that we can just run `npm run license', and we don't have to type in those options.

Also, I think it should just write to THIRD-PARTY-LICENSES.csv, and git will detect if there are differences or not; no need to run diff manually.

The downside to this is handling the unknown/unlicensed cases, but in our discussion you said that one that was unknown, you couldn't determine even by going to the repo, so I think we just leave it like that.

Reply:
Hmmm, it seems that this way there will be no way of permanently saying a certain dependency is X license. 
Not sure how to work around that. 
One option that might help might be to run the license checker on travis/circle and fail the build if the license checker seems to be changing the number of lines in `UPDATED-LICENSES.csv`

Reply:
added a test on circleci to check if the package.json and package-lock.json have been changed in comparison to what's on develop. If they have, check the third-party-licenses.csv is also updated compared to develop. Fail if not.